src/Microsoft.Extensions.Logging.Testing/NullLoggerOfT.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Logging.Testing
+{
+    public class NullLogger<T> : ILogger<T>
```

Comment:
I didn't do this because none of the other loggers use inheritance between `<T>` and non-generic